### PR TITLE
Fix WCPay in core texts and promo slug

### PR DESCRIPTION
**Pull request by ilyasfoo**
This PR fixes promo texts to reflect the actual WCPay promo applied in backend (100% discount instead of 50%).

### Screenshots

image

#### Testing the changed texts
- In your local, edit `should_add_the_menu` to return true in `src/Features/WcPayWelcomePage.php` to simulate treatment experiment (currently the experiment is disabled for February)
- Go to payments welcome page and observe the correct texts in banner and header card

#### Testing promo note action
- In payments welcome screen, click on "Install"
- Browse your database in `wp_wc_admin_notes` table, make sure a note with the name `wcpay-promo-2022-3-incentive-100-off` is created and its status is `actioned`

### CODE REVIEW

**Review by chihsuan — APPROVED** [review]
Test well and LGTM. 🚢

**Comment by ilyasfoo** [comment]
Thank you for reviewing, @chihsuan!

I think we should wait for feedback on 1604-gh-Automattic/woocommerce-payments-server before merging to confirm if our new slug is okay to use.

**Comment by ilyasfoo** [comment]
I think this is safe to merge now, the feedback on mentioned PR does would not affect this PR.

**Pull request merged**
